app/views/clients/files/_file_row.haml

```diff
@@ -27,16 +27,24 @@
   .row
     .col-3
       - if file.visible_in_window
-        %em This file is visible in the window
```

Comment:
style changes before and after:

<img width="1288" alt="Screen Shot 2022-08-05 at 12 35 37 PM" src="https://user-images.githubusercontent.com/5333982/183122104-2d627865-1d3e-4c75-801f-a0223c81ee05.png">

-> 

<img width="1287" alt="Screen Shot 2022-08-05 at 12 35 05 PM" src="https://user-images.githubusercontent.com/5333982/183122132-a2421204-37ec-4df0-b528-a60f20d8235e.png">